admin/sql/CreateTables.sql

```diff
@@ -400,6 +392,12 @@ CREATE TABLE country_area ( -- replicate (verbose)
     area                INTEGER -- PK, references area.id
 );
 
+CREATE TABLE deleted_entity (
```

Comment:
I don’t think it makes sense to have separate tables per entity type here, because there is no added type safety (we can’t have foreign keys to the relevant `artist` etc. tables because the entries have been removed from those).